js/quotechew.js

```diff
@@ -113,8 +130,42 @@ var RE_SECTION_DELIM = /^[_-]{6,}$/;
 
 var RE_LIST_BOILER = /mailing list$/;
 
+/**
+ * Non-localized hard-coded heuristic to help us detect whether a line is the
+ * lead-in to a quote block.  This is only checked against the last content line
+ * preceding a quote block.  This only results in styling guidance, so if it's
+ * too aggresive or insufficiently aggressive, the worst that happens is the
+ * line will be dark gray instead of black.  (For now!  Moohoohahahahahaha.)
+ */
 var RE_WROTE_LINE = /wrote/;
 
+/**
```

Comment:
Well, I don't know that it actually *needs* to, but bugs will be filed if it does not since that's a particular favorite test of QA.